src/main/java/coordinatecalculator/domain/Line.java

```diff
@@ -0,0 +1,38 @@
+package coordinatecalculator.domain;
+
+import coordinatecalculator.domain.parent.Figure;
+
+import java.util.List;
+
+public class Line implements Figure {
+    private static final String LINE_RESULT_MESSAGE = "두 점 사이 거리는 ";
+
+    private final Points points;
+
+    Line(final Points points) {
+        this.points = points;
+    }
+
+    @Override
+    public double calculateResult() {
+        List<Point> points = this.points.getSortedPoints();
+        Point p1 = points.get(0);
+        Point p2 = points.get(1);
+
+        int x = Math.abs(p2.getX() - p1.getX());
+        int y = Math.abs(p2.getY() - p1.getY());
+
+        return Math.sqrt(x * x + y * y);
+    }
+
+    @Override
+    public String makeResult() {
+        StringBuilder sb = new StringBuilder(LINE_RESULT_MESSAGE);
+        return sb.append(calculateResult()).toString();
```

Comment:
```suggestion
        return return LINE_RESULT_MESSAGE + calculateResult();
```
성능이 문제가 되지 않는다면 더 간결하게 가는건 어떨까요?

Reply:
추천해주신 코드로 변경했습니다. 추천해주신 코드가 객체 생성이 적어서 더 좋은 것 같습니다.